nhs/data/handling.py

```diff
@@ -103,4 +106,68 @@ def read_spreadsheets(
             lambda x: x[0], placeholder_matches(files, file_dir_pattern, ["key"])
         )
 
-    return {key: val for key, val in zip(keys, map(reader, files))}
+    return {key: reader(file, sheet_id) for key, file in zip(keys, files)} if extension == "xlsx" else {
+        key: reader(file) for key, file in zip(keys, files)
+    }
+
+
+@log_entry_exit(level="INFO")
+def standarise_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
+                     identification: str, abbreviation_column_name: str,
+                     long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+    """
+    Process the column names of a polar Lazy frame dictionary to make them more readable.
+        1. Expand abbreviated names.
+        2. Converting to lower case.
+    If any parameters are None, this function returns prev_csv_pl_df without change.
+    The filename in both dictionaries are checked or filtered to be passed.
+
+    Parameters:
+    --------
+    param1 : dict[str, pl.LazyFrame]
+        df_dict: A dictionary containing census data as value and filename as key or None.
+    param2 : [pl.LazyFrame | None]
+        census_metadata: Census metadata lazy frame with one standard sheet or None.
+        Blank files won't change prev_csv_pl_df.
+    param3 : str
+        identification: Identify different files in df_dict. Could not be None
+    param4 : str
+    abbreviation_column_name: column name of abbreviated names. Could not be None
+    param5 : str
+    long_column_name: column name of expanded names. Could not be None
+
+    Returns:
+    --------
+    dict[str, pl.LazyFrame]
+        A dictionary with updated column names based on the provided mapping.
+
+    Examples
+    --------
+        Assume we have the following files in the directory `"path/to/csv_files/"`:
+            - 2021Census_G01_AUS_AUS.csv
+            - 2021Census_G02_AUS_AUS.csv....
+        and "path/to/xlsx_files/":
+            - Metadata_2021_GCP_DataPack_R1_R2.xlsx
+            - 2021Census_geog_desc_1st_2nd_3rd_release.xlsx
+        Standard file in second sheet of Metadata_2021_GCP_DataPack_R1_R2.xlsx
+        "name of XLSX" : {"name of sheet1" : <dataframe>, "name of sheet2": <dataframe> ...}
+        By using read_spreadsheets to get dictionary containing dataFrames.
+        >>> def standarise_names({"files_identify1_with_abbreviated_names": <LazyFrame>,
+        >>>"files_identify2_with_abbreviated_names": <LazyFrame>},
+        >>>"standard_sheet1's" <LazyFrame>, "identification", "abbreviation_column_name", "Long_column_name")
+        {"file_identify1_with_expanded_names": <LazyFrame>, "file_identify2_with_expanded_names": <LazyFrame>}
```

Comment:
Also forgot to mention in the first PR, can you wrap the code in \`\`\`python ... \`\`\`? So VSCOde can render it as code

Reply:
up to you